appendices/VK_EXT_buffer_device_address.txt

```diff
@@ -66,7 +66,9 @@ None
 
 === Issues
 
-None
+1) Where is VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_BUFFER_ADDRESS_FEATURES_EXT and VkPhysicalDeviceBufferAddressFeaturesEXT?
+
+*RESOLVED*: They were renamed as VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_BUFFER_DEVICE_ADDRESS_FEATURES_EXT and VkPhysicalDeviceBufferDeviceAddressFeaturesEXT accordingly for consistency.
```

Comment:
Add something like "the old names are still available, for compatibility".

Reply:
That somewhat makes the question invalid, if the old names are still available...

Reply:
Add some words for compatibility in the new commit.